src/svgSprite.ts

```diff
@@ -568,7 +568,7 @@ export const svgSprite = (): string => `
     </symbol>
 
     <symbol viewBox="0 0 24 24" id="svg-more-info-filled">
-      <path d="M12 2C6.5 2 2 6.5 2 12C2 17.5 6.5 22 12 22C17.5 22 22 17.5 22 12C22 6.5 17.5 2 12 2ZM13 16C13 16.6 12.6 17 12 17C11.4 17 11 16.6 11 16V12C11 11.4 11.4 11 12 11C12.6 11 13 11.4 13 12V16ZM12 9C11.4 9 11 8.6 11 8C11 7.4 11.4 7 12 7C12.6 7 13 7.4 13 8C13 8.6 12.6 9 12 9Z" fill="#00639A"/>
```

Comment:
We might want to pass `fill="#00639A"` to the other places we're using `<Svg id="more-info-filled" />` since they are probably expecting that default (though it's possible `currentColor` works for them with no change). I see it in these places:

- src/shared/components/Banners/Banner/Banner.const.tsx
- src/modules/Builder/pages/BuilderApplet/BuilderApplet.utils.tsx
- src/modules/Builder/pages/BuilderActivityFlow/BuilderActivityFlow.utils.tsx
- src/modules/Builder/pages/BuilderActivity/BuilderActivity.utils.tsx

Reply:
src/shared/components/Banners/Banner/Banner.const.tsx
<img width="121" alt="Screenshot 2024-03-22 at 10 05 36 AM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1165936/e75298d6-340c-4669-b2de-dac92caa8211">

src/modules/Builder/pages/BuilderApplet/BuilderApplet.utils.tsx
<img width="226" alt="Screenshot 2024-03-22 at 10 06 24 AM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1165936/0f95b1de-66e7-44b5-8453-465c9d4ae434">

src/modules/Builder/pages/BuilderActivityFlow/BuilderActivityFlow.utils.tsx
<img width="329" alt="Screenshot 2024-03-22 at 10 07 59 AM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1165936/b4145f4b-e25f-48db-91fc-c95131f9debf">

src/modules/Builder/pages/BuilderActivity/BuilderActivity.utils.tsx
<img width="267" alt="Screenshot 2024-03-22 at 10 08 43 AM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1165936/156f7c07-8afc-4fa3-b922-5179928ad99b">

Ha ok! So I found all the places, and they're all in the navigation/banner. I checked and it seems like they continue to be the correct color. So you can ignore this comment!